spec/Section 6 -- Execution.md

```diff
@@ -620,6 +626,9 @@ the type system to have a specific input type.
 At each argument position in an operation may be a literal {Value}, or a
 {Variable} to be provided at runtime.
 
+Any _request error_ raised during {CoerceArgumentValues()} should be treated
+instead as an _execution error_.
+
```

Comment:
this is fine for now. Ideally anywhere we raise a request error is definitely a request error.

Is the case this is covering against per-type input coercion?